clayui.com/src/utils/getSection.js

```diff
@@ -122,10 +108,16 @@ const getSection = (data) => {
 		);
 	});
 
-	const rootElements = elements.filter((path) => path.isRoot);
+	const rootElements = elements.filter(path => path.isRoot);
+
+	const parentPaths = elements.reduce((acc, {navigationParent}) => {
```

Comment:
I pulled this outside of `toSectionItem` to reduce the amount of times it would be called. Otherwise we would have to check on every call to that function

Reply:
Smart 👍 I was going with what I knew could work.